src/app/zap-templates/zcl/data-model/chip/tls-certificate-management-cluster.xml

```diff
@@ -36,7 +36,7 @@ Git: 1.4-534-g3214b3502
   </struct>
 
   <cluster apiMaturity="provisional">
-    <domain name="General"/>
+    <domain name="General">General</domain>
```

Comment:
webrtc provider does not have this duplicate name and content for the domain. Why did we get this change?

Reply:
Something changed with alchemy 😬 It's a benign change, because the value is read from both, but I was going to submit a PR to only use value or content, not both